deployment_scripts/appimage/build_appimage.sh

```diff
@@ -168,24 +177,26 @@ build_frontend() {
     popd || handle_error
 }
 
-build_appimage() {
-    log_message "Building AppImage"
-    appimage-builder --recipe monkey_island_builder.yml --log DEBUG --skip-appimage
+add_monkey_icon() {
+	unlink "$APPDIR"/python.png
+	mkdir -p "$APPDIR"/usr/share/icons
+	cp "$REPO_MONKEY_SRC"/monkey_island/cc/ui/src/images/monkey-icon.svg "$APPDIR"/usr/share/icons/infection-monkey.svg
+	ln -s "$APPDIR"/usr/share/icons/infection-monkey.svg "$APPDIR"/infection-monkey.svg
+}
 
-    # There is a bug or unwanted behavior in appimage-builder that causes issues
-    # if 32-bit binaries are present in the appimage. To work around this, we:
-    #   1. Build the AppDir with appimage-builder and skip building the appimage
-    #   2. Add the 32-bit binaries to the AppDir
-    #   3. Build the AppImage with appimage-builder from the already-built AppDir
-    #
-    # Note that appimage-builder replaces the interpreter on the monkey agent binaries
-    # when building the AppDir. This is unwanted as the monkey agents may execute in
-    # environments where the AppImage isn't loaded.
-    #
-    # See https://github.com/AppImageCrafters/appimage-builder/issues/93 for more info.
-    download_monkey_agent_binaries
+add_desktop_file() {
+	unlink "$APPDIR"/python3.7.9.desktop
+	cp ./infection-monkey.desktop "$APPDIR"/usr/share/applications
+	ln -s "$APPDIR"/usr/share/applications/infection-monkey.desktop "$APPDIR"/infection-monkey.desktop
+}
 
-    appimage-builder --recipe monkey_island_builder.yml --log DEBUG --skip-build
+add_apprun() {
+	cp ./AppRun "$APPDIR"
+}
+
+build_appimage() {
+    log_message "Building AppImage"
+    ARCH="x86_64" appimagetool "$APPDIR"
```

Comment:
So now we're using appimagetool instead of appimage-builder? Update the readme accordingly